typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -575,11 +575,19 @@ class ContextFunder {
   }
 
   private getDesiredBalanceForRole(chain: ChainName, role: Role): BigNumber {
-    const desiredBalanceEther =
-      role === Role.Kathy && this.desiredKathyBalancePerChain[chain]
-        ? this.desiredKathyBalancePerChain[chain]
-        : this.desiredBalancePerChain[chain];
-    let desiredBalance = ethers.utils.parseEther(desiredBalanceEther ?? '0');
+    let desiredBalanceEther: string;
+    if (role === Role.Kathy) {
+      const desiredKathyBalance = this.desiredKathyBalancePerChain[chain];
+      if (desiredKathyBalance === undefined) {
+        logger.warn({ chain }, 'No desired balance for Kathy, not funding');
+        desiredBalanceEther = '0';
+      } else {
+        desiredBalanceEther = this.desiredKathyBalancePerChain[chain];
+      }
+    } else {
+      desiredBalanceEther = this.desiredBalancePerChain[chain];
+    }
+    let desiredBalance = ethers.utils.parseEther(desiredBalanceEther);
```

Comment:
if role is not kathy, could we still get undefined desiredBalanceEther? ie do we still need this 
```suggestion
    let desiredBalance = ethers.utils.parseEther(desiredBalanceEther ?? '0');
```